src/terminfo/mod.rs

```diff
@@ -283,6 +289,87 @@ pub struct TerminfoTerminal<T> {
     ti: TermInfo,
 }
 
+#[cfg(unix)]
```

Comment:
You don't actually have to duplicate this entire implementation; you can just `cfg` out the specific function.